docs/handlers.md

```diff
@@ -0,0 +1,118 @@
+# Handler plugin development
+
+The purpose of Handler plugins is to generate some kind of output from log entries.
+This can mean writing some representation of the entries into a log file, or sending
+the raw entries to an external service for further processing, or anything else, really.
+
+## The `HandlerPlugin` interface
+
+Each Handler plugin must implement the `HandlerPlugin` interface, which extends
+the generic [`Plugin` interface] and adds the following methods and properties:
+
+ - `public log(entry: LogEntry): Promise<void> | void`: This method will be called from
+   `Logger.add()`. Whatever a handler wants to do with log entries is up to the handler.
+ - `public runTask?<R>(cb: () => R): R`: This optional method can be implemented by
```

Comment:
Taky mi to prijde nejasne. "Takze kdyz tohle nenaimplementuju, tak mi nefungujou tasky?"

Reply:
Tyjo, co je nejasnyho na "can be implemented by handlers _which need to integrate_ with Debugr tasks"? plus je to dal vysvetleny v samostatny sekci `Tasks` o kus niz